main.go

```diff
@@ -150,3 +161,20 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func gitCommit() (string, error) {
+	bi, ok := debug.ReadBuildInfo()
+	if !ok {
+		return "", errors.New("could not read build info")
+	}
+
+	const vcsRevisionKey = "vcs.revision"
```

Comment:
I'll move this const to line 166

Reply:
+1

Reply:
Why?

Reply:
is just a good practice to declare const at the beginning of a func or pkg.